python_transport/wirepas_gateway/utils/argument_tools.py

```diff
@@ -541,6 +541,33 @@ def add_filtering_config(self):
             ),
         )
 
+    def add_rtc_settings(self):
```

Comment:
Your new service should have its own arugment parser that has nothing to do with transport one